testsuites/syncgateway/performance/kill_gateload.py

```diff
@@ -1,10 +1,22 @@
 import os
-import subprocess
+import sys
+from libraries.provision.ansible_runner import AnsibleRunner
 
 
 def kill_gateload():
-    os.chdir("performance_tests/ansible/playbooks")
-    subprocess.call(["ansible-playbook", "-i", "../../../provisioning_config", "kill-gateload.yml"])
+    try:
+        cluster_config = os.environ["CLUSTER_CONFIG"]
+    except KeyError:
+        print ("Make sure CLUSTER_CONFIG is defined and pointing to the configuration you would like to provision")
+        sys.exit(1)
+
+    print("Running perf test against cluster: {}".format(cluster_config))
+    ansible_runner = AnsibleRunner(cluster_config)
+
+    status = ansible_runner.run_ansible_playbook("kill-gateload.yml")
+    if status != 0:
+        print("Killing gateload returned non-zero status: {}.  Most likely it was no longer running".format(status))
```

Comment:
In what cases will gateload not be finished at this point? Why do we have to kill gateload? I thought is ended on its own after the runtime_ms.